.github/workflows/ci.yml

```diff
@@ -49,24 +49,29 @@ jobs:
 
   test:
     name: Run Molecule tests.
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     strategy:
       fail-fast: true
       matrix:
-        distro:
-          - ubuntu2004
+        image:
+          - ghcr.io/hifis-net/ubuntu-systemd:20.04
 
     steps:
       - name: Check out the codebase.
         uses: actions/checkout@v3
 
+      - name: Enable lingering required for podman and systemd in GH Actions.
+        run: |  # Compare with https://github.com/eriksjolund/user-systemd-service-actions-workflow/blob/efe872924fd2dd35bb482544126ce751303e14c2/README.md
+          sudo loginctl enable-linger $UID
+          sleep 1
+
       - name: Prepare the job environment.
         uses: ./.github/workflows/prepare-action
 
       - name: Run Molecule tests.
-        run: pipenv run molecule test
+        run: XDG_RUNTIME_DIR=/run/user/$UID pipenv run molecule test
         env:
-          MOLECULE_DISTRO: ${{ matrix.distro }}
+          MOLECULE_DISTRO: ${{ matrix.image }}
 
   release:
```

Comment:
Do we want to use `ubuntu-22.04` for the release job too?

Reply:
Yes, that totally makes sense. This is changed now.